Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN set -x \
   # Install cypress dependencies & JRE (required by Jenkins)
   && echo 'deb http://ftp.debian.org/debian jessie-backports main' >> /etc/apt/sources.list \
   && apt-get update \
-  && apt-get install -y xvfb libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 lsof \
+  && apt-get install -y xvfb libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 lsof coreutils \
```

Comment:
why is this needed? Could you split it up to an extra commit so we can, later on, identify why it was added?

Reply:
Coreutils is added so we can use the timeout function.